rollup.config.js

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* global __dirname */
+
+/**
+ * External dependencies
+ */
+import { resolve as resolvePath, dirname } from 'path';
+import resolve from '@rollup/plugin-node-resolve';
+import commonjs from '@rollup/plugin-commonjs';
+import { babel } from '@rollup/plugin-babel';
+import { terser } from 'rollup-plugin-terser';
+import svgr from '@svgr/rollup';
+import filesize from 'rollup-plugin-filesize';
+import css from 'rollup-plugin-import-css';
+import url from '@rollup/plugin-url';
+import json from 'rollup-plugin-json';
+import license from 'rollup-plugin-license';
+import workspacesRun from 'workspaces-run';
+
+/**
+ * Internal dependencies
+ */
+
+const plugins = [
```

Comment:
We also need a rollup plugin to clean up the dist folder files when bundling. 

After a few builds I started getting the following error

```bash
npm notice === Tarball Details ===
npm notice name:          @googleforcreators/story-editor
npm notice version:       0.1.202201271454
npm notice filename:      @googleforcreators/story-editor-0.1.202201271454.tgz
npm notice package size:  14.9 MB
npm notice unpacked size: 46.9 MB
npm notice shasum:        fb74cfc732ad86c4b018be5e1224cd46a9b7ae8e
npm notice integrity:     sha512-+0SiTYz3Q2+9R[...]bP2o7eCD5PfYQ==
npm notice total files:   1108
npm notice
npm ERR! code E413
npm ERR! 413 Payload Too Large - PUT http://localhost:4873/@googleforcreators%2fstory-editor - request entity too large

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/sayedtaqui/.npm/_logs/2022-01-27T14_54_19_044Z-debug.log
```


It's because it keeps adding new files without removing the old ones. We may use [rollup-plugin-delete](https://www.npmjs.com/package/rollup-plugin-delete) and clear `dist` and `dist-module` folders

```js
del({
    targets: ['dist/*', 'dist-module/*']
  }),
```

Reply:
Will add